enterprise/server/remote_execution/containers/firecracker/BUILD

```diff
@@ -163,7 +163,7 @@ go_test(
         "--executor.firecracker_enable_merged_rootfs=true",
         "--executor.firecracker_enable_uffd=true",
         "--executor.enable_local_snapshot_sharing=true",
```

Comment:
@maggie-lou  I re-enabled this since this test is marked with `tags = ["manual"]` anyway